src/View/AutoSegmentation/ButtonInputBox.py

```diff
@@ -0,0 +1,102 @@
+from collections.abc import Callable
+
+from PySide6.QtWidgets import QWidget, QPushButton, QVBoxLayout, QLabel, QLineEdit, QHBoxLayout
+from PySide6.QtCore import QSize
+
+from src.View.StyleSheetReader import StyleSheetReader
+from src.View.WindowBarIconSetup import setup_window
+
+
+class ButtonInputBox(QWidget):
+    """
+    A Message Box which has the with labels and actions to run when buttons are clicked
```

Comment:
This has labels called "with" ?
What is this going to be used for?  I get the feeling it's useful and that it prepackages some common functionality.
Why would a programmer want to use this?